### PR TITLE
Fix version output from `get-version` action

### PR DESCRIPTION
**Pull request by LitoMore**
Fixes https://github.com/simple-icons/simple-icons/actions/runs/12458105634/job/34773364133

I used `xargs` to trim spaces and tabs.

### CODE REVIEW

File: .github/actions/get-version/action.yml [hunk 1]
```diff
@@ -12,5 +12,5 @@ runs:
     - id: get-version
       shell: sh
       run: |
-        version="$(grep version -m 1 -i package.json | sed 's/[ \",:version]//g')"
+        version="$(grep version -m 1 -i package.json | sed 's/[ \",:version]//g' | xargs)"
```
Comment (mondeja) [inline .github/actions/get-version/action.yml:15]: This has been broken by #12380. Please, use this instead:

```sh
grep version -m 1 -i package.json | cut -d '"' -f4
```

Your fix looks like a patch over an outdated command.

**Review by mondeja — CHANGES_REQUESTED** [review]

**Review by mondeja — APPROVED** [review]

**Comment by uncenter** [comment]
Why aren't we using jq for extracting the version field? Would be `jq .version package.json -r` I think.

**Comment by mondeja** [comment]
> Why aren't we using jq for extracting the version field? Would be just `jq .version package.json -r`, and works regardless of formatting or location of the key in the package.json. jq is included by default on GitHub Actions runners iirc.

I always write no-dependency UNIX-Posix Shell syntax by default, which is totally portable to all shells. This `sed` shit was legacy code added on v4, with `cut` would have worked regardless the tabs change.

It seems to me that if we depend on jq, we depend on another thing which is always a bad decission. I don't really care so much because probably GH will depend on jq forever, but even depending on GH itself seems wrong to me.

**Comment by mondeja** [comment]
Tested performance and jq is 10 times slower:

```bash
$ time for i in $(seq 1 100); do grep version -m 1 -i package.json | cut -d '"' -f4; done
real	0m0,447s
user	0m0,215s
sys	0m0,522s

$ time for i in $(seq 1 100); do jq .version package.json -r; done
real	0m3,958s
user	0m3,586s
sys	0m0,388s
```

**Comment by uncenter** [comment]
```
$ hyperfine 'jq .version package.json -r'
Benchmark 1: jq .version package.json -r
  Time (mean ± σ):       1.7 ms ±   2.4 ms    [User: 0.9 ms, System: 1.1 ms]
  Range (min … max):     0.5 ms …  19.0 ms    129 runs
```

We are talking about 2 milliseconds. There is zero point in benchmarking this.

**Comment by uncenter** [comment]
```
$ hyperfine 'grep version -m 1 -i package.json | cut -d \'"\' -f4' 'jq .version package.json -r'
Benchmark 1: grep version -m 1 -i package.json | cut -d '"' -f4
  Time (mean ± σ):       2.6 ms ±   0.4 ms    [User: 3.3 ms, System: 0.8 ms]
  Range (min … max):     2.0 ms …   5.1 ms    523 runs

Benchmark 2: jq .version package.json -r
  Time (mean ± σ):       2.3 ms ±   0.5 ms    [User: 1.1 ms, System: 0.5 ms]
  Range (min … max):     1.5 ms …   4.8 ms    490 runs

Summary
  jq .version package.json -r ran
    1.17 ± 0.31 times faster than grep version -m 1 -i package.json | cut -d '"' -f4
```

**Comment by mondeja** [comment]
> We are talking about 2 milliseconds.

What does hyperfine? Why the result is so different against a Bash for loop?

**Comment by mondeja** [comment]
Ah I see what it does... just another "efficient box"...

> There is zero point in benchmarking this.

Just to aggregate something anecdotic, consider the other arguments.

**Comment by LitoMore** [comment]
`cut` is clear enough for trimming text.

If we need to operate JSON frequently, I would recommend `jq` because it is more intuitive.

But here we only have one line to operate. The `jq` is too much.

**Comment by uncenter** [comment]
Fwiw there is also `npm pkg get version` that we could use. It is marginally slower but makes it readable and consistent.

**Comment by mondeja** [comment]
> Fwiw there is also `npm pkg get version` that we could use. It is marginally slower but makes it readable and consistent.

`npm pkg get version` prints the version wrapped by `"` characters that need to be stripped anyways. The full command would be something like `npm pkg get version | tr -d '"'` which needs npm as dependency and is not very much readable anyways.

**Pull request merged**